client/web/app/outline_server_repository/vpn.ts

```diff
@@ -27,6 +29,8 @@ export interface StartRequestJson {
   client: string;
   // First hop used by the legacy Electron code.
   firstHop: string;
+  // Type of connection the config will create
+  connectionType: ConnectionType;
```

Comment:
We don't need the connection type in the start request. Please remove.

Reply:
done